sympy/functions/special/beta_functions.py

```diff
@@ -120,3 +121,83 @@ def _eval_conjugate(self):
 
     def _eval_rewrite_as_gamma(self, x, y, **kwargs):
         return self._eval_expand_func(**kwargs)
+
+
+###############################################################################
+########################### INCOMPLETE BETA  FUNCTION #########################
+###############################################################################
+
+
+
+class betainc(Function):
+    r"""
+    The incomplete beta function is a generalization of the beta function,
+
+    .. math::
+        \mathrm{B}_{z}(a,b) := \int^{z}_{0} t^{a-1} (1-t)^{b-1} \mathrm{d}t.
+
+    Incomplete beta function satisfies properties like:
+
+    .. math::
+        \mathrm{B}_{1}(a,b) = \mathrm{B}(a,b) \\
+
+    See Also
+    ========
+    Gamma function, Upper incomplete gamma function, Lower incomplete gamma function,
+    Polygamma function, Log Gamma function, Digamma function, Trigamma function.
```

Comment:
See here for formatting this section:
https://docs.sympy.org/latest/documentation-style-guide.html#see-also-section